cds/base/bundles.py

```diff
@@ -39,11 +40,13 @@
         "react": "0.11.2",
         "jsx-requirejs-plugin": "0.4",
         "requirejs-plugins": "latest",
-        "es5-shim": "latest"
+        "es5-shim": "latest",
+        "requirejs-hogan-plugin": "latest"
```

Comment:
Thank you for the PR! Only one small thing IMHO we could remove this line from here because is already in https://github.com/inveniosoftware/invenio/blob/c7c9b3e68c7d577e687763ac6b4a1e9ba564d5d3/invenio/base/bundles.py WDYT? :)